openapi-cli/src/test/java/io/ballerina/openapi/CodeGeneratorTest.java

```diff
@@ -475,6 +476,70 @@ public void testCodeGenerationWithoutSchemas() {
         }
     }
 
+    @Test(description = "Test code generation when no schemas given in service")
+    public void testCodeGenerationWithoutSchemasService() {
+        final String serviceName = "no_schema";
+        String definitionPath = RES_DIR.resolve("no_schema.yaml").toString();
+        CodeGenerator generator = new CodeGenerator();
+        try {
+            generator.generateService(definitionPath, serviceName, resourcePath.toString(),  filter, false);
+            if (Files.exists(resourcePath.resolve("no_schema_service.bal")) &&
+                    Files.notExists(resourcePath.resolve("types.bal"))) {
+                Assert.assertTrue(true);
+            } else {
+                Assert.fail("Empty types.bal file has been generated");
+            }
```

Comment:
Can't we directly use the `assertTrue(boolean condition, String message)` here to avoid using `Assert.assertTrue(true)` statements? Let's refactor all the other places as well.